documentation/assemblies/assembly-securing-kafka.adoc

```diff
@@ -0,0 +1,46 @@
+// This assembly is included in the following assemblies:
+//
+// // assembly-securing-access.adoc
+
+[id='assembly-securing-kafka-{context}']
+= Securing access to Kafka brokers
+
+To establish secure access to Kafka brokers, you configure and apply:
+
+* A `Kafka` resource to create listeners with a specified authentication type
+* A `KafkaUser` resource to access the Kafka brokers securely through the listeners
+
+Configure the `Kafka` resource to set up:
+
+* Listener authentication
+* Network policies that restrict access to Kafka listeners
+* Kafka authorization
+* Super users for unconstrained access to brokers
+
+Authentication is configured independently for each listener.
+Authorization is always configured for the whole Kafka cluster.
+
+The Cluster Operator creates the listeners and sets up the cluster and client certificate authority (CA) certificates to enable authentication within the Kafka cluster.
+
+You can replace the certificates generated by the Cluster Operator by link:{BookURLUsing}#installing-your-own-ca-certificates-str[installing your own certificates^].
+You can also link:{BookURLUsing}#kafka-listener-certificates-str[configure your listener to use a Kafka listener certificate managed by an external Certificate Authority^].
+Certificates are available in PKCS #12 format (.p12) and PEM (.crt) formats.
+
+Configure the `KafkaUser` resource to set up:
+
+* Client authentication to match the listener authentication
+* Client authorization to match the Kafka authorization
```

Comment:
Can we make it clear that the `KafkaUser` configures the authentication and authorization which the specific user will be allowed to use versus the enabled mechanisms in the Kafka CR? Otherwise I think this will be confusing.

Maybe we could also mention quotas? It is also slightly related to security.

Reply:
Changed to:

> Configure the `KafkaUser` resource to set up:
> 
> * Authentication to match the enabled listener authentication
> * Authorization to match the enabled Kafka authorization
> * Quotas to control the use of resources by clients
> 
> You configure authentication and authorization that the client is allowed to use.